src/join.njk

```diff
@@ -6,9 +6,9 @@
     <span class="column">
       <p>We love new members! Whether you're brand new to the sport and want to try it out, or super-experienced and want to train for your next world record, you want you to feel welcome at Christchurch Freediving Club.</p>
       <p>Everyone has to attend an induction, even if you've been diving for years, you've taken a course, or you're a member of another club, see our <a href="start.html">starting page</a>. You must be 18 or older to do an induction or join the club.</p>
-      <p>Once you've done an induction, you can join the club. You'll need to fill out the membership form below and pay the club fees. Our fees are $180 per calendar year ($90 for students). Then you can attend our <a href="train.html">training sessions</a>, trips, and other events. Alternatively, you can pay $10 per session for casual training (but you still need to do an induction).</p>
+      <p>Once you've done an induction, you can join the club. You'll need to fill out the membership form below and pay the club fees. Our fees are $120 per calendar year ($90 for students). Then you can attend our <a href="train.html">training sessions</a>, trips, and other events. Alternatively, you can pay $10 per session for casual training (but you still need to do an induction).</p>
```

Comment:
Did you want to update the student price too, or keep it as is?

Reply:
Let's keep it as it is for now, please.